website/static/js/addProjectPlugin.js

```diff
@@ -176,12 +176,13 @@ var AddProject = {
                             m('label[for="projectName].f-w-lg.text-bigger', 'Title'),
                             m('input[type="text"].form-control.project-name', {
                                 onkeyup: function(ev){
-                                    if (ev.which === 13) {
-                                         ctrl.add();
-                                    }
                                     var val = ev.target.value;
+                                    var validTitle = val.trim().length > 0;
+                                    if (ev.which === 13 && validTitle) {
```

Comment:
Got it! I will add the title validity check on top of `add()` and leave `onkeyup` event handler as was.